src/main/java/org/ibp/api/java/germplasm/GermplasmListService.java

```diff
@@ -66,4 +66,6 @@ void addGermplasmEntriesToList(
 
 	void addGermplasmListEntriesToAnotherList(String cropName, String programUUID, Integer destinationListId, Integer sourceListId,
 		SearchCompositeDto<GermplasmListDataSearchRequest, Integer> searchComposite);
+
+	void removeGermplasmEntriesFromList(Integer germplasmListId, List<Integer> listDataIds);
```

Comment:
Same as in MW, use S`earchCompositeDto<GermplasmListDataSearchRequest, Integer> searchComposite` as 2nd parameter